package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qless-js",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
This should probably be a minor version jump since clients may be relying on the current exceptions being caught.

Reply:
Good idea